Bump selenium-webdriver

oliverguenther opened the pull request:
Fixes Firefox startup for v43.0 on my Mac.

oliverguenther committed:
Bump selenium-webdriver

Fixes Firefox startup for v43.0 on my Mac.

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://oliverguenther-bump-selenium-webdriver.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://oliverguenther-bump-selenium-webdriver.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

oliverguenther committed:
Ensure the viewport layer is visible for Firefox

Firefox doesn't seem to be able to select the element otherwise, and
simply selects some text instead.

oliverguenther commented:
Closing since there appear to be breaking changes in the event handling that I can't solve now.
